zuul.d/project-templates.yaml

```diff
@@ -76,6 +76,9 @@
               - name: github.com/ansible-collections/community.aws
               - name: github.com/ansible-collections/community.general
         - ansible-test-cloud-integration-aws-py36
```

Comment:
And don't forget to drop the mother/father job :-)

Reply:
yeah I just caught that  :)

Reply:
I mean, you should not include the `ansible-test-cloud-integration-aws-py36` job anymore.

Reply:
OH!  I misunderstood, thanks!